### PR TITLE
Post with highlights from tidyverse 1.2.* update

### PR DESCRIPTION
**Pull request by batpigandme**
Not wed to image choice at all— the 🐈🐈🐈🐈 playing various string instruments were _not_ 200 × 1,000 px friendly.

### CODE REVIEW

File: content/articles/tidyverse_1.2.0.Rmd [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  author: Gerd Altmann
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+library("tidyverse")
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+The release of [tidyverse 1.2.0](https://github.com/tidyverse/tidyverse/releases)
```
Comment (hadley) [inline content/articles/tidyverse_1.2.0.Rmd]: I think it is now tradition to say "We are pleased to announce..."

File: content/articles/tidyverse_1.2.0.Rmd [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+```{r tidyverse_install, eval = FALSE}
+install.packages("tidyverse")
+```
+
+## Changes in tidyverse membership
+
+### Addition of the reprex package
+
+Though it's been a mainstay tidyverse [Help](https://www.tidyverse.org/help/),
```
Comment (hadley) [inline content/articles/tidyverse_1.2.0.Rmd]: "Help" -> "of help"?  Not quite sure what you're trying to say here

File: content/articles/tidyverse_1.2.0.Rmd [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Though it's been a mainstay tidyverse [Help](https://www.tidyverse.org/help/), 
+1.2.0 marks the official addition of [reprex](http://reprex.tidyverse.org/) 
+(short for **repr**oducible **ex**ample) to the tidyverse. 
+
+The goal of a reprex is to package your problematic code in such a way that 
+other people can run it and feel your pain. Think of every request for a 
+reprex as a plea to _"help me help you!"_
+
+This update includes reprex **Addins**, accessible from the RStudio drop-down
```
Comment (hadley) [inline content/articles/tidyverse_1.2.0.Rmd]: I'd say something more like: "This now means that you can access the handy reprex **addins**, ..."

File: content/articles/tidyverse_1.2.0.Rmd [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+(short for **repr**oducible **ex**ample) to the tidyverse. 
+
+The goal of a reprex is to package your problematic code in such a way that 
+other people can run it and feel your pain. Think of every request for a 
+reprex as a plea to _"help me help you!"_
+
+This update includes reprex **Addins**, accessible from the RStudio drop-down 
+menu.
+
+![](https://i.imgur.com/PxRzbTq.png)
```
Comment (hadley) [inline content/articles/tidyverse_1.2.0.Rmd]: These should go in `content/articles` IIRC

File: content/articles/tidyverse_1.2.0.Rmd [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+packages (for string, and categorical variable manipulation, respectively) have 
+joined [ggplot2](http://ggplot2.tidyverse.org/), [dplyr](http://dplyr.tidyverse.org/), [tidyr](http://tidyr.tidyverse.org/), [readr](http://readr.tidyverse.org/), 
+[purrr](http://purrr.tidyverse.org/), and [tibble](http://tibble.tidyverse.org/) 
+to make up the core tidyverse. 
+
+Running `library(tidyverse)` attaches these core packages in the current R session.
+On attach, tidyverse now makes better use of horizontal space, printing packages
+and versions in two columns (see below). Additionally, packages that have 
+already been attached are not be printed, and development versions are highlighted
+in red. The startup message can now be suppressed with
```
Comment (hadley) [inline content/articles/tidyverse_1.2.0.Rmd]: The startup message -> "You can suppress this message using `suppressPackageStartupMessages()` or by setting `options(...)`

File: content/articles/tidyverse_1.2.0.Rmd [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+While a comprehensive list of changes can be found in the [release notes](https://github.com/tidyverse/tidyverse/releases), other highlights 
+include:
+
+ * `tidyverse_conflicts()` now prints all conflicts involving at least one
+ tidyverse package (including intra-tidyverse conflicts, omitted in previous
+ versions)
+ 
+ * Addition of a `tidyverse_logo()` function.
+
+![](https://i.imgur.com/D6TJvPn.gif)
```
Comment (hadley) [inline content/articles/tidyverse_1.2.0.Rmd]: Can you please also include an acknowledgements section that thanks:

```
[Derek Chiu](https://github.com/dchiu911), [Matthew Lincoln](https://github.com/mdlincoln), [Michael Toth](https://github.com/michaeltoth) and [Serhat Cevikel](https://github.com/serhatcevikel)
```

And I'll share my script with you to use in the future. It probably needs to go into some sort of repo somewhere

File: content/articles/tidyverse_1.2.0.Rmd (hunk 2 above)
Comment (batpigandme) [reply content/articles/tidyverse_1.2.0.Rmd]: It's clearly a cry for help!

File: content/articles/tidyverse_1.2.0.Rmd (hunk 4 above)
Comment (batpigandme) [reply content/articles/tidyverse_1.2.0.Rmd]: 👍 The only example I could think of in this dir was Lucy's post, and her [images](https://github.com/tidyverse/tidyverse.org/tree/master/static/images/lucy-internship) are in `static/images/lucy-internship`. I have no preference, just didn't know SOP.

File: content/articles/tidyverse_1.2.0.Rmd [hunk 7]
```diff
@@ -0,0 +1,75 @@
+---
+title: tidyverse 1.2.0
+slug: tidyverse-1.2.0
```
Comment (jennybc) [inline content/articles/tidyverse_1.2.0.Rmd]: Does the underscore in the filename versus the hypen here in the slug explain why the post isn't showing up in the netlify preview (?).

Comment (batpigandme) [reply content/articles/tidyverse_1.2.0.Rmd]: Just changed it to see. Thanks, Jenny.

**Comment by hadley** [comment]
Oh I think the reason it's not showing up is that the filename doesn't include the date in it

**Comment by hadley** [comment]
SUCCESS!!

**Comment by jimhester** [comment]
I wonder either the gifs could not repeat, or stay longer at the last frame. The first in particular is hard to see what the output is before it restarts.

**Comment by batpigandme** [comment]
@jimhester copy that, I've got them on `0.5` for final frame delay, but I could make that longer, or have it go just once (that can get weird, though, since that once-around doesn't necessarily start when you get to it) 🎞 😭

**Pull request merged**